### DISCUSSION TITLE
Allow music for all rides

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17759] [Feature requests]
I'd like Unlimited Ride Music added to Maze, Merry Go Round, and any other ride with music restrictions

**Comment by karst** [comment]
The "Unlock Operating Limits" cheat allows this

**Comment by 733737** [comment]
Now that we have custom music, it would be nice to allow more music in parks and with different rides. It doesn't effect game balance in any way
